src/Microsoft.DocAsCode.Metadata.ManagedReference/ExtractMetadataWorker.cs

```diff
@@ -389,6 +390,67 @@ private static ExtractMetadataInputModel ValidateInput(ExtractMetadataInputModel
             }
         }
 
+        private async Task<IEnumerable<IMethodSymbol>> GetAllExtensionMethodsAsync(IEnumerable<Project> projects)
+        {
+            List<INamespaceSymbol> namespaces = new List<INamespaceSymbol>();
+            foreach (var project in projects)
+            {
+                var compilation = await project.GetCompilationAsync();
```

Comment:
double compile, and what about extension methods in dependency assembly (not in source code)?